functions/mqpublish.go

```diff
@@ -61,15 +61,11 @@ func Checkin(ctx context.Context, wg *sync.WaitGroup) {
 }
 
 func checkin() {
-
+	// check/update host settings; publish if changed
 	if err := UpdateHostSettings(); err != nil {
-		logger.Log(0, "failed to update host settings -", err.Error())
+		logger.Log(0, "checkin", err.Error())
 		return
 	}
-
-	if err := PublishGlobalHostUpdate(models.HostMqAction(models.CheckIn)); err != nil {
```

Comment:
why is this removed?
if checkin never occurs all the nodes will go into error state...

Reply:
UpdateHostSetttings calls PublishGlobalHostUpdate so leaving this would result in two identical messages being published.

Reply:
Testing
as per the Abhishek comments we are facing first time warning issue and after some time it was showing error status also please check attachment

screenshot: 
1.https://prnt.sc/TRsSgZF4tEPL
2.https://prnt.sc/5YhDLi-6mZ4E
3.https://prnt.sc/OZJD52D4CDm6

Reply:
> UpdateHostSetttings calls PublishGlobalHostUpdate so leaving this would result in two identical messages being published.
UpdateHostSetttings will only publish message when there is something that has changed, this won't accommodate `checkin` functionality

Reply:
removed PublishGlobalHostUpdate from UpdateHostSetting